src/patient-lab-dashboard/test-results/test-results.tsx

```diff
@@ -216,27 +216,41 @@ const TestResults: React.FC<TestResultProps> = ({
       )
   }
 
+  const getItems = (test, datatype) => {
+    const items = []
+    if (datatype === 'Boolean') {
+      items.push({name: {name: 'True'}}, {name: {name: 'False'}})
+    } else if (datatype === 'Coded') {
+      const answers = test.answers
+      items.push(...answers)
+    }
+    return items
+  }
+
+  const getDropdownItemToString = (test, labResult) => data => {
+    const itemName = data.name.name
+    if (
+      labResult.get(test.uuid)?.abnormal === true &&
+      labResult.get(test.uuid)?.value === itemName
+    ) {
+      return <span style={{color: 'red'}}>{itemName}</span>
+    }
+    return itemName
+  }
+
   const renderInputField = (test, index) => {
     if (test) {
-      const items = []
       const datatype = test.datatype.name
-      if (datatype === 'Boolean' || datatype === 'Coded') {
-        if (datatype === 'Boolean')
-          items.push({name: {name: 'True'}}, {name: {name: 'False'}})
-        else if (datatype === 'Coded') {
-          const answers = test.answers
-          for (let answer of answers) {
-            items.push(answer)
-          }
-        }
-        return (
-          <div className={styles.testresultinputfield}>
+      const items = getItems(test, datatype)
+      return (
+        <div className={styles.inputFieldWithCheckbox}>
+          {datatype === 'Boolean' || datatype === 'Coded' ? (
             <Dropdown
               titleText={getTestNameWithUnits(test)}
               id="answers-list-dropdown"
               title="answers list"
               items={items}
-              itemToString={data => data.name.name}
+              itemToString={getDropdownItemToString(test, labResult)}
```

Comment:
passing labResult as a parameter is not required since it is a state variable

Reply:
The parameter list for getDropdownItem has been updated, thanks @abinaya-u for the suggestion.